documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc

```diff
@@ -133,11 +133,9 @@ Depending on how you apply OAuth 2.0 authentication, and the type of authorizati
 <6> (Only applicable when using `introspectionEndpointUri`) The authorization server may be configured or implemented in such a way to not provide any identifiable information in an Introspection Endpoint response. In order to obtain the user ID, you can configure the URI of the `userinfo` endpoint as a fallback. The `userNameClaim`, `fallbackUserNameClaim`, and `fallbackUserNamePrefix` settings are applied to the response of `userinfo` endpoint.
-<7> Set this to `false`to disable the OAUTHBEARER mechanism on the listener. At least one of PLAIN or OAUTHBEARER has to be enabled. Default is `true`.
-<8> Set to `true` to enable PLAIN authentication on the listener, which is supported by all clients on all platforms. The Kafka client must enable the PLAIN mechanism and set the `username` and `password`. PLAIN can be used to authenticate either by using the OAuth access token, or the OAuth `clientId` and `secret` (the client credentials). The behavior is additionally controlled by whether `tokenEndpointUri` is specified or not. Default is `false`.
-If `tokenEndpointUri` is specified and the client sets `password` to start with the string `$accessToken:`, the server interprets the password as the access token and the `username` as the account username. Otherwise, the `username` is interpreted as the `clientId` and the `password` as the client `secret`, which the broker uses to obtain the access token in the client's name.
-If `tokenEndpointUri` is not specified, the `password` is always interpreted as an access token and the `username` is always interpreted as the account username, which must match the principal id extracted from the token. This is known as 'no-client-credentials' mode because the client must always obtain the access token by itself, and can't use `clientId` and `secret`.
-<9> Additional configuration for PLAIN mechanism to allow clients to authenticate by passing `clientId` and `secret` as `username` and `password` as described in the previous point. If not specified the clients can authenticate over PLAIN only by passing an access token as `password` parameter.
+<7> Set this to `false` to disable the OAUTHBEARER mechanism on the listener. At least one of PLAIN or OAUTHBEARER has to be enabled. Default is `true`.
+<8> Set to `true` to enable PLAIN authentication on the listener, which is supported for clients on all platforms.
+<9> Additional configuration for the PLAIN mechanism. Clients can authenticate by passing a client ID and secret as a `username` and `password`. Default is `false`. If not specified, clients can authenticate over PLAIN only by passing an access token as the `password` using an `$accessToken:` prefix.
```

Comment:
I'm not sure what 9 refers to without seeing a rendered doc, but the only setting I can think of here is `tokenEndpointUri` which is not true / false.

```suggestion
<9> Additional configuration for the PLAIN mechanism. Clients can authenticate by passing a client ID and secret as a `username` and `password`, or by passing account name and access token with an `$accessToken:` prefix. If not specified, clients can authenticate over PLAIN only by passing an unprefixed access token as the `password`.
```

Reply:
Yes. It's the callout for the tokenEndpointUri. 
Maybe this is enough:

> Additional configuration for the PLAIN mechanism. If specified, clients can authenticate over PLAIN by passing an access token as a `password` using an `$accessToken:` prefix.

Reply:
I don't know, maybe. I like to repeat myself in different places because usually we don't read docs from start to finish but go look for a solution. Here being reminded that how you pass access token depends on whether this is configured or not.

Reply:
Agreed. But this a callout. I don't think we need to be verbose. We have the options covered in a couple of other places.
I've added a link in the _Before you start_ pf this procedure that goes to the _Oauth 2.0 authentication mechanisms_ section as reference